services/QuillLMS/Gemfile.lock

```diff
@@ -511,19 +511,19 @@ GEM
       rack
     rack-proxy (0.6.5)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    rails (5.0.7.2)
-      actioncable (= 5.0.7.2)
-      actionmailer (= 5.0.7.2)
-      actionpack (= 5.0.7.2)
-      actionview (= 5.0.7.2)
-      activejob (= 5.0.7.2)
-      activemodel (= 5.0.7.2)
-      activerecord (= 5.0.7.2)
-      activesupport (= 5.0.7.2)
+    rack-test (1.1.0)
```

Comment:
Pin `rack-test` to `gem 'rack-test', '~> 0.6.3'` to avoid an unnecessary major version bump (I tested this on your branch and it resolves). Otherwise, looks good!